apps/client/src/components/Layouts/DefaultLayout.tsx

```diff
@@ -16,10 +16,14 @@ function DefaultLayout({
   matomoPageView,
 }: DefaultLayoutProps) {
   const { location } = useHistory();
+  const [oldLocation, setLocation] = useState("");
```

Comment:
Should the default not be `location.pathname` to set it in case of reload?

Reply:
We set a location.pathname it doesn't track the originalPageview, it needs to be empty at start.